test/anttasks/org/apache/cassandra/anttasks/JdkProperties.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.anttasks;
+
+import org.apache.tools.ant.Project;
+import org.apache.tools.ant.Task;
+
+public class JdkProperties extends Task
+{
+
+    public void execute()
+    {
+        Project project = getProject();
+        project.setNewProperty("java.version." + project.getProperty("ant.java.version").replace("1.", ""), "true");
+        project.setNewProperty("use-jdk" + project.getProperty("ant.java.version").replace("1.", ""), "true");
```

Comment:
do we still need this?

Reply:
Yeah, the replace was needed only for JDK8. I will remove that part